recipes/magic-class/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "magic-class" %}
+{% set version = "0.5.19" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/magic-class-{{ version }}.tar.gz
+  sha256: f0f5730fc5cc65d808333a2106a253fe5fff97e85240b3c1cc73f49e0ade6c2b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - macro-kit >=0.3.5
+    - magicgui >=0.3.4
+    - python >=3.8
+    - qtpy >=1.10.0
+
+test:
+  imports:
+    - magicclass
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+    - pyqt
+
+about:
+  home: https://pypi.org/project/magic-class/
```

Comment:
```suggestion
  home: https://github.com/hanjinliu/magic-class
```

Reply:
should we add dev_url and put the github liunk in there?

Reply:
🤷🏼